markdown/events/2022/training-october-2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: nf-core Training - October 2022
+subtitle: A virtual Nextflow and nf-core training event
+type: workshop
+start_date: '2022-10-03'
+start_time: '03:00 CEST'
+end_date: '2022-10-05'
+end_time: '21:30 CEST'
+location_name: Virtual
+import_typeform: true
```

Comment:
Don't think that we need this - pretty sure it was just when we were embedding typeform surveys in the page before.
```suggestion
```